.xo-config.json

```diff
@@ -2,5 +2,18 @@
   "plugins": ["prettier"],
   "extends": ["prettier"],
   "envs": ["node", "es2022"],
-  "space": 4
+  "space": 4,
+  "rules": {
+    "max-params": [
+      "warn",
+      {
+        "max": 5
+      }
+    ],
+    "n/file-extension-in-import": "off",
+    "n/prefer-global/process": "off",
+    "no-await-in-loop": "off",
+    "no-new": "off",
+    "unicorn/prefer-node-protocol": "off"
```

Comment:
I don't mind disabling XO rules - but could you possibly briefly outline why these rules need to be disabled? Inconvenient though the linting rules can sometimes be, I _usually_ find them to be appropriate - but as I say, it's OK to disable them so long as we think it's appropriate.

Reply:
Earlier on, I was having issues with getting my code to pass linting, with most of the errors coming from the new AWS related code. As I figured the vle integrations repo must use similar code, I figured that I'd copy the .xo-config from the vle integrations repo, which I cut down to just these few, which solved my issues at the time. 

I've just checked through to see which ones I still need, as of now, the only one I need to keep is no-new, so I'll cut down these new rules to just "no-new" :)

Reply:
For reference, I need the no-new rule so that I can use the "new" keyword when creating the cdk stack.

e.g. new ManageBlackboardUsersStack(app, "ManageBlackboardUsersStack", {.....